failer.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 The Knative Authors
```

Comment:
Is this same as the file in config/failer/failer.yaml?

We probably don't want this at the root also.